src/main/java/edu/hm/hafner/analysis/descriptor/AcuCobolDescriptor.java

```diff
@@ -0,0 +1,59 @@
+package edu.hm.hafner.analysis.descriptor;
+
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.AcuCobolParser;
+
+/**
+ * A Descriptor for the AcuCobol warnings.
+ *
+ * @author Lorenz Munsch
+ */
+public class AcuCobolDescriptor implements Descriptor {
+
+    private static final String ID = "AcuCobol";
+
+    /**
+     *
+     * Name to identify the warning.
+     *
+     * @return the identification string
+     */
+    @Override
+    public String getName() {
+        return ID;
```

Comment:
This is not relevant for your use case in IntelliJ. But in the warnings plugin (and in my GitHub autograding action) it would make sense to define an ID and a human readable name. An ID should not have white space while the name can. So I would prefer to use a `name` as you are already using is and an additional `id` with only ASCII lowercase and `-` (can be used as URL as well). (The names are stored in https://github.com/jenkinsci/warnings-ng-plugin/blob/master/plugin/src/main/resources/io/jenkins/plugins/analysis/warnings/Messages.properties)